sources/input_to_text_tree_mocks/mock_10.c

```diff
@@ -0,0 +1,27 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   mock_10.c                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/14 23:07:09 by inikulin          #+#    #+#             */
+/*   Updated: 2024/10/22 20:47:05 by inikulin         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "input_to_text_tree_mock_internal.h"
+
+void	mock_10_tree(t_treenode *cur)
+{
+	ft_treenode_insert_child_idx_s_dup(cur, "pwd", 0);
+}
+
+/* returns 1 on success - to allow condition usage outside */
```

Comment:
do you live comments like here directly in github? how do you do it? :)

Reply:
No idea. I just write them in vim while writing the code

Reply:
![image](https://github.com/user-attachments/assets/d35125b3-2534-47e1-9769-e5f0e0b34aac)
what changes though?